OverlayPlugin.Core/EventSources/FFXIVRequiredEventSource.cs

```diff
@@ -369,43 +376,133 @@ public override void SaveConfig(IPluginConfig config)
 
         public override void Start()
         {
-            this.timer.Change(0, this.Config.UpdateInterval * 1000);
+            this.timer.Change(0, 10);
```

Comment:
Do you think we need party updates this frequently?

Reply:
This matches what base `FFXIV_ACT_Plugin` is doing (unless I misread the decompiled code).

Reply:
As for timing, I don't think we need every 10ms either, I'm open for suggestions on the pacing of this. I just don't think it should be tied to the `Config.UpdateInterval` timing.

Maybe 100ms or 250ms makes more sense?

Reply:
Yeah I think either of those is fine.  I'm trying to think of use cases for this event, and most things I think of would be fine knowing about party members on a granularity of multiple seconds even.

If you were writing UI for the party, probably Dalamud plugin is a better bet here to overwrite the existing party/alliance list, and so "extremely responsive updates for UI" doesn't feel like a goal here.

Reply:
I think the new setup makes more sense?